torch/testing/_internal/common_methods_invocations.py

```diff
@@ -164,6 +164,7 @@ def __init__(self,
                  supports_sparse=False,  # supported for sparse
                  check_batched_grad=True,  # check batched grad when doing gradcheck
                  check_batched_gradgrad=True,  # check batched grad grad when doing gradgradcheck
+                 is_linear_map=False,  # non-linear mappings on sparse tensors coalesce the inputs
```

Comment:
This is good but let's make this comment more general:

"whether the op is a linear map, f(a + b) = f(a) + f(b), f(c * a) = c * f(a)"

Reply:
To support sparse sample inputs (below) this should add sample_inputs_func_sparse=None, like sample_inputs_func on line 160

Reply:
@mruberry - Implemented the above suggestions